src/gui/src/CommonStrings.h

```diff
@@ -0,0 +1,212 @@
+/*
+ * Hydrogen
+ * Copyright (C) 2021 The hydrogen development team <hydrogen-devel@lists.sourceforge.net>
+ *
+ * http://www.hydrogen-music.org
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY, without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ *
+ */
+
+#ifndef COMMON_STRINGS_H
+#define COMMON_STRINGS_H
+
+#include <QString>
+#include <core/Object.h>
+
+/** A container class to collect all translatable strings at one place
+	in order to allow for the reusage. This will (some day) make
+	translation work faster.
+
+	Initially I wanted the whole thing as well as the getter methods
+	to be static. But Qt requires the class to be initialized for the
+	translation engine to take effect.*/
```

Comment:
> (eg. by adding convenience methods in Object) 

I also thought about introducing a `GUIObject` which would be just a wrapper around `H2Core::Object` in `src/gui/src` in order to avoid mixing core and gui stuff. It could have a `getCommonStrings()` protected method (and potentially some debugging function showing basic QWidget properties, like focus, size, tooltip etc.)